starlette/routing.py

```diff
@@ -196,7 +196,12 @@ def matches(self, scope: Scope) -> typing.Tuple[Match, Scope]:
                     matched_params[key] = self.param_convertors[key].convert(value)
                 path_params = dict(scope.get("path_params", {}))
                 path_params.update(matched_params)
-                child_scope = {"endpoint": self.endpoint, "path_params": path_params}
+                child_scope = {
+                    "endpoint": self.endpoint,
+                    "path_params": path_params,
+                    "route": self.path,
```

Comment:
I'd probably rather we were including the `Route` itself, since there's other information on there too that the developer might potentially want access to.